db/migrate/20171219154111_convert_skip_rule_to_uuid.rb

```diff
@@ -0,0 +1,23 @@
+class ConvertSkipRuleToUuid < ActiveRecord::Migration
```

Comment:
Good call. Do you know if there's a way to make UUID the new default on rails 4? Might be worth that config change if it can be done. Otherwise we should put something in the backlog to make sure we set that up on rails 5 as I'm pretty sure it _can_ be done there.

Reply:
http://www.mccartie.com/2015/10/20/default-uuid's-in-rails.html

I will add.